src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+

```

Comment:
Let's add the realization to all override methods in classes that extend Machine 
for example: 
```suggestion
 System.out.println("Bulldozer started working!");
```